src/main/webapp/app/core/auth/account.service.ts

```diff
@@ -138,15 +138,15 @@ export class AccountService {
         return this.authenticated;
     }
 
-    isIdentityResolved(): boolean {
```

Comment:
When the user logs out, the userIdentity is set to null.
This makes this method dangerous in my opinion as it will return true in this case even though userIdentity is null.